bundles/org.openhab.binding.modbus.sunspec/src/main/java/org/openhab/binding/modbus/sunspec/internal/parser/InverterModelParser.java

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.modbus.sunspec.internal.parser;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openhab.binding.modbus.sunspec.internal.SunSpecConstants;
+import org.openhab.binding.modbus.sunspec.internal.dto.InverterModelBlock;
+import org.openhab.io.transport.modbus.ModbusRegisterArray;
+
+/**
+ * Parses inverter modbus data into an InverterModelBlock
+ *
+ * @author Nagy Attila Gabor - Initial contribution
+ *
+ */
+@NonNullByDefault
+public class InverterModelParser extends AbstractBaseParser implements SunspecParser<InverterModelBlock> {
+
+    @Override
+    public InverterModelBlock parse(ModbusRegisterArray raw) {
```

Comment:
please remove the blank lines in this method

Reply:
okay, removed those.